pkg/lib/v0_2_0/events.go

```diff
@@ -473,3 +482,169 @@ func ToKeptnEvent(event cloudevents.Event) (models.KeptnContextExtendedCE, error
 
 	return keptnEvent, nil
 }
+
+// CreateStartedEvent takes a parent event (e.g. .triggered event) and creates a corresponding .started event
+func CreateStartedEvent(source string, parentEvent models.KeptnContextExtendedCE, eventData interface{}) (*models.KeptnContextExtendedCE, error) {
+	if err := validateParentEvent(parentEvent); err != nil {
+		return nil, err
+	}
+
+	startedEventType, err := ReplaceEventTypeKind(*parentEvent.Type, "started")
+	if err != nil {
+		return nil, fmt.Errorf("unable to create '.started' event for parent event %s: %w", parentEvent.ID, err)
+	}
+
+	if eventData == nil {
+		commonEventData := EventData{}
+		if err := parentEvent.DataAs(&commonEventData); err != nil {
+			logrus.Errorf("unable to retrieve event data from parent event %s: %s", parentEvent.ID, err.Error())
```

Comment:
Nit: 
```
logrus.Errorf("unable to retrieve event data from parent event %s: %v", parentEvent.ID, err))
```

Reply:
or actually, should we instead return an error here? failing to extract the `EventData` means that no project/stage/service will be included in the event, meaning that Keptn will not be able to correlate it to a sequence

Reply:
I didn't want to change the behavior of the methods in this stage of development, it was like this originally, so I just added logging